### ISSUE TITLE
Sync Trakt to Stremio instead

### ISSUE THREAD

**Issue by aliyss** [aliyss/trakt-stremio-import#1]

**Comment by ghost** [comment]
Could you make trakt up next episides to stremio catalog, there's rss feed in trakt as well for up next episodes.

**Comment by aliyss** [comment]
@Zero0Q Should have some time this weekend will look into it.

**Comment by ghost** [comment]
OMG if you can make this happen I would be the happiest man alive, stremio is great app the only thing missing is catalog with up next when a new episode is out there's no way to know unless going in to trakt. I tried to install your app but can't find a way to do it, when do npm install and npm start I get error msg that a folder is missing. Could you walk me through installation process please?

**Comment by aliyss** [comment]
Does stremio not actually show upcoming episodes in the catalog?

Also I'm making the whole thing better with [syncribullet](https://github.com/aliyss/syncribullet)
So that might actually ease stuff up anyways.

**Comment by ghost** [comment]
No it doesn't, it's a shame it's a great app.

**Comment by ghost** [comment]
Can syncribullet add catalogs from trakt like next watching episodes, even from SIMKL if you could add to import lists that would be awesome.

**Comment by ghost** [comment]
Stremio up next is only missing on tv app...

**Comment by ghost** [comment]
> @Zero0Q Should have some time this weekend will look into it.
I hope you had a wonderful weekend. Did you had any time to look for trakt up next over the weekend?

**Comment by aliyss** [comment]
yep... but not working stable yet. Issue is I have to take all the code from here and move it to syncribullet.

**Comment by SeanHaz** [comment]
So glad you shared this. Exactly what I was looking for. 

I've made some stuff like this before, ugly but got the job done, I never posted it publicly. You've convinced me that it's the wrong choice. You never know who it might help :)
